docs/pages/access-controls/sso.mdx

```diff
@@ -328,7 +328,11 @@ spec:
   # in the Web interface
   display: "Okta"
 
-  acs: https://teleport-proxy.example.com:3080/v1/webapi/saml/acs
+  # enables/disables idp-initiated saml login
+  allow_idp_initiated: false
```

Comment:
Since false is the default, it's probably unlike to see this snippet in a connector, as people will only add `allow_idp_initiated` if they want to enable it. Should we change this example to `true` instead? (and probably add a note about the security implications)

Reply:
Hmm, I'm a bit wary of default-on since I'm afraid some may just copy-paste this or not read it thoroughly. I added this here for clarity so that astute readers may enable it if they so choose. Do you think it'd be worth it to change it to true? I'm 50/50 I feel like.

Reply:
Could you pair a `true` value with the flag that still allows local user login, and then have them optionally remove that option later once the configuration is complete?